akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventSourcedBehaviorImpl.scala

```diff
@@ -122,9 +122,7 @@ private[akka] final case class EventSourcedBehaviorImpl[Command, Event, State](
     // stashState outside supervise because StashState should survive restarts due to persist failures
     val stashState = new StashState(ctx.asInstanceOf[ActorContext[InternalProtocol]], settings)
 
-    /**
-     * This method ensures that the MDC is set before we use the internal logger
-     */
+    // This method ensures that the MDC is set before we use the internal logger
     def internalLogger() = {
```

Comment:
This is not a method but a nested function inside of a method so scaladoc/javadoc on it is not valid